tools/regionalgam/ab-index.R

```diff
@@ -4,8 +4,10 @@ args = commandArgs(trailingOnly=TRUE)
 source(args[1])
 
 
-tryCatch({input = read.table(args[2], header=TRUE,sep=" ")},finally={input = read.table(args[2], header=TRUE,sep=",")})
-pheno = read.table(args[3], header=TRUE,sep="	")
+# import file
+input = fread(args[2])
+# retransform to data.frame to allow the calculation by the fonctions from the package
```

Comment:
functions

Reply:
error from my side ! Thank you @bgruening